sdks/python/container/build.gradle

```diff
@@ -27,13 +27,26 @@ description = "Apache Beam :: SDKs :: Python :: Container"
 // "./github.com/apache/beam/sdks/go"
 resolveBuildDependencies.dependsOn ":beam-sdks-go:build"
 
+configurations {
+  dockerDependency
+}
+
 dependencies {
   golang {
     // TODO(herohde): use "./" prefix to prevent gogradle use base github path, for now.
     // TODO(herohde): get the pkg subdirectory only, if possible. We spend mins pulling cmd/beamctl deps.
     build name: './github.com/apache/beam/sdks/go', dir: project(':beam-sdks-go').projectDir
     test name: './github.com/apache/beam/sdks/go', dir: project(':beam-sdks-go').projectDir
   }
+
+  dockerDependency project(path: ":beam-sdks-python", configuration: "distConfig")
+}
+
+task copyDockerfileDependencies(type: Copy) {
+  from configurations.dockerDependency
+  rename "apache-beam.*\\.tar.gz", "apache-beam.tar.gz"
```

Comment:
Doesn't the distConfig already publish a single artifact named apache-beam.tar.gz

Reply:
Indeed, after this PR it does.